data/sql/updates/pending_db_world/Eyes of Skettis.sql

```diff
@@ -0,0 +1,6 @@
+-- Veil Reskk
+DELETE FROM `gameobject` WHERE `id` = 185200;
+UPDATE `gameobject` SET `spawntimesecs`=0 WHERE `guid`=26086 AND `id`=185200;
+--  Veil Shienor
+DELETE FROM `gameobject` WHERE `id` = 182505;
+UPDATE `gameobject` SET `spawntimesecs`=0 WHERE `guid`=26087 AND `id`=185201;
```

Comment:
This won't work. Because you are deleting the ID's, it will delete ALL spawns. So the update query will do nothing.

Always Delete with ID and GUID

Reply:
Good point. I'll update it in a bit when I'm free again